uploader.json

```diff
@@ -239,6 +244,11 @@
             "pattern": [],
             "symbolic_name": "hostname"
         },
+        {
+            "command": "/bin/hostname -I",
+            "pattern": [],
+            "symbolic_name": "ip_addresses"
+        },
```

Comment:
Should this be "command": "/usr/bin/hostname -I" instead of "command": "/bin/hostname -I" to match the spec in core?